xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/resources/liveTableLiveDataConfiguration.json

```diff
@@ -13,6 +13,7 @@
     "propertyDescriptors": [
       {"id": "doc.name", "type": "String", "editable": false, "displayer": "link"},
       {"id": "doc.title", "type": "String", "displayer": "link"},
+      {"id": "doc.hidden", "type": "Boolean", "displayer": "boolean"},
```

Comment:
The ``Boolean`` type should use the ``boolean`` displayer by default so we can omit it, right?